README.md

```diff
@@ -53,7 +53,7 @@ Questions can be asked by raising an `Issue`.
 - #### Add your name to the section ([Hactoberfest community](https://github.com/EddieHubCommunity/hacktoberfest-practice#hacktoberfest-community)) that is headed with your first initial. Then, add your name in alphabetical order of the second letter in your name. If the second letters are the same, order it in alphabetical order of the third, and so on. Next to it, add the link to your github username page.
 
 - For example ,
-  `- [Full Name](https://github.com/your-username)`
+  `- [Moeen Ul Islam](https://github.com/moeen-ul-islam)`
```

Comment:
Hey Moeen you don't have to change it here. If you scroll down you will see **E** section there add you name in correct alphabetical order. If you need any help let us know

Reply:
I am new to all this, please bear with me.
How to add my name in alphabetical order?

Reply:
@Moeen-Ul-Islam Don't worry, add your name `- [Moeen Ul Islam](https://github.com/moeen-ul-islam)` at line `386` below **Mitanshi Kshatriya**'s name :slightly_smiling_face:

Reply:
So do I make another PR for that or is there any other way ?

Reply:
Please change this line back to `- [Full Name](https://github.com/your-username)`